test/models/due_date_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
```

Comment:
Please write RSpec tests under the `spec` folder.

Reply:
Done writing and refactoring the same. Thank you for guiding the team. We appreciate it.